lib/core/presentation/pages/discover/discover_page/discover_page.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:app/core/presentation/widgets/common/appbar/appbar_logo.dart';
+import 'package:app/core/presentation/widgets/common/appbar/lemon_appbar_widget.dart';
+import 'package:app/core/presentation/widgets/discover/discover_card.dart';
+import 'package:app/core/presentation/widgets/event/event_discover_item.dart';
+import 'package:app/core/presentation/widgets/poap/hot_badge_item.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/router/app_router.gr.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:sliver_tools/sliver_tools.dart';
+
+@RoutePage()
+class DiscoverPage extends StatelessWidget {
+  const DiscoverPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    return Scaffold(
+      backgroundColor: Theme.of(context).colorScheme.primary,
+      appBar: LemonAppBar(
+        title: t.discover.discover,
+        leading: const AppBarLogo(),
+        actions: [
+          GestureDetector(
+            onTap: () {
+              AutoRouter.of(context).navigate(const ChatListRoute());
+            },
+            child: ThemeSvgIcon(
+              color: Theme.of(context).colorScheme.onSurface,
+              builder: (filter) => Assets.icons.icChatBubble.svg(
+                colorFilter: filter,
+              ),
+            ),
+          ),
+        ],
+      ),
+      body: CustomScrollView(
+        slivers: [
+          SliverPadding(
+            padding: EdgeInsets.symmetric(horizontal: Spacing.xSmall),
+            sliver: const _DiscoverCards(),
+          ),
+          SliverPadding(
+            padding: EdgeInsets.symmetric(vertical: Spacing.xSmall),
+          ),
+          const _HotBadgesNearYou(),
```

Comment:
Consider should we define as folder and put these widget inside folder

```
discover
    - UpcomingEvents
    - HotBadgeNearYou
```

Reply:
Yes, it should be put separate file. For future testing purpose

Reply:
yes, for big parts of view, this should be in separated file